src/communication/websocket/listener.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include "communication/websocket/listener.hpp"
+
+namespace communication::websocket {
+namespace {
+void LogError(boost::beast::error_code ec, const std::string_view what) {
```

Comment:
```suggestion
void LogError(boost::beast::error_code ec, std::string_view what) {
```

Reply:
`what` doesn't change throughout the function definition, why not have it as a const?

Reply:
Sry, you are right